CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Cromwell Change Log
 
+## 38 Release Notes
+
+### Bug fixes
+- Fixed an issue that could cause Cromwell to consume disk space unnecessarily when using zipped dependencies
```

Comment:
"`could`" seems optimistic here

Reply:
Deliberate wording; if someone is running Cromwell on a Linux distro that e.g. cleans up `/tmp` every 24 hours rather than on every reboot, they will have never seen this bug